src/main/java/teammates/ui/pagedata/FeedbackSubmissionEditPageData.java

```diff
@@ -281,4 +281,4 @@ public boolean getIsResponsePresent() {
         }
         return false;
     }
-}
+}
```

Comment:
Add a new line at the EOF

Reply:
Thank you :smile: Seemed like I went to delete it :laughing: